spec/requests/reviewable_notes_controller_spec.rb

```diff
@@ -0,0 +1,323 @@
+# frozen_string_literal: true
+
+RSpec.describe "Reviewable Notes" do
+  let(:admin) { Fabricate(:admin) }
+  let(:moderator) { Fabricate(:moderator) }
+  let(:user) { Fabricate(:user) }
+  let(:reviewable) { Fabricate(:reviewable_flagged_post) }
+
+  describe "#create" do
+    context "when user is staff" do
+      before { sign_in(admin) }
+
+      it "creates a new reviewable note successfully" do
+        post "/reviewables/#{reviewable.id}/notes.json",
+             params: {
+               reviewable_note: {
+                 content: "This is a test note",
+               },
+             }
+
+        expect(response.status).to eq(200)
+
+        json = response.parsed_body
+        expect(json["content"]).to eq("This is a test note")
+        expect(json["user"]["id"]).to eq(admin.id)
+        expect(json["user"]["username"]).to eq(admin.username)
+
+        # Verify note was actually created in database
+        note = ReviewableNote.last
+        expect(note.content).to eq("This is a test note")
+        expect(note.user).to eq(admin)
+        expect(note.reviewable).to eq(reviewable)
+      end
+
+      it "works for moderators too" do
```

Comment:
```suggestion
      it "creates a new reviewable note successfully as a moderator" do
```

Suggestion to make the test description more specific to the behaviour we are testing for here.